<launch>
  <node type="waypoint_server_node.py" name="waypoint_server" pkg="waypoint_server" output="screen">
    <param name="threshold_distance" value="1"/>
    <param name="gps_topic" value="/gps"/>
    <param name="odom_topic" value="/odom"/>
    <param name="publish_displacement_from_wp" value="True"/>
  </node>
</launch>
